### DISCUSSION TITLE
PDF export does not show font out of PowerPoint

### DISCUSSION THREAD

**Discussion by BTultra** [rsms/inter#754] [Q&A] [answered]
hello,
We are using the font in a lot of PowerPoint Presentations, but discovered that PowerPoint does not transfer the font when printig or exporting a presentation to a PDF file.

I found out that the problem is lies in the integrability or embedding value: this is set to "installable", but should be set to "printable" in order for Powerpoint to be able to hand the font over when eporting to PDF.

Is there any possibility to change this or a way i can change this by myself?

**Comment by kenmcd** [comment, marked as answer]
Installable also includes the ability to Print.
It basically means - no restrictions.

I had no problems exporting from PowerPoint 2019 to PDF.
Below is an image of the PDF output.

![Inter-Display-PPT-to-PDF](https://github.com/user-attachments/assets/3f843c14-2f49-4429-b6ba-2aa128bcb500)

Note: Office applications (PowerPoint, Word, etc.) currently cannot export variable fonts to PDFs.
Which is why in my example above you only see the regular weight for the variable.

Printing variable fonts works as expected - see printed image below.
![Microsoft PowerPoint - Inter Display pptx_20](https://github.com/user-attachments/assets/4bc5d81a-48c2-43ac-ae54-c3365befa0ca)


So the font permissions is not the problem.

You may be having issues due to font name conflicts.
I have only the v4.000 variable and statics from this repo installed.
They have different family names so they do not conflict.

The Google Fonts version of Inter variable family name will conflict with the statics from this repo, and with older versions of Inter.
Office 365 did have some older versions of Inter (not sure if they still do).
So make sure there is no Inter folder in a the CloudFonts folder if you have O2021, O2024, or O365.

So only install the fonts you actually need.
Or just the fonts released from this repo.
Or just the fonts released from GF.

**Reply by BTultra** [reply]
Ah, thanks for clarification! Then i will use static fonts (one weight per file).
